tethys_apps/cli/__init__.py

```diff
@@ -195,19 +217,48 @@ def tethys_command():
     services_create_sd.add_argument('-n', '--name', required=True, help='A unique name for the Service', type=str)
     services_create_sd.add_argument('-c', '--connection', required=True, type=str,
                                     help='The connection of the Service in the form '
-                                         '"<username>:<password>@<protocol>//<host>:<port>"')
-    services_create_sd.add_argument('-p', '--public-endpoint', required=False, type=str,
+                                    '"<username>:<password>@<protocol>//<host>:<port>"')
+    services_create_sd.add_argument('-p', '--public-endpoint', type=str,
                                     help='The public-facing endpoint, if different than what was provided with the '
-                                         '--connection argument, of the form "<host>:<port>"')
-    services_create_sd.add_argument('-k', '--apikey', required=False, type=str,
+                                    '--connection argument, of the form "<host>:<port>"')
+    services_create_sd.add_argument('-k', '--apikey', type=str,
                                     help='The API key, if any, required to establish a connection.')
     services_create_sd.set_defaults(func=services_create_spatial_command)
 
+    # tethys services create dataset
+    services_create_dataset = services_create_subparsers.add_parser('dataset',
+                                                                    help='Create a CKAN/HydroShare Dataset Service.')
+    services_create_dataset.add_argument('-n', '--name', required=True, help='A unique name for the Service', type=str)
+    services_create_dataset.add_argument('-t', '--type', required=True, type=str, choices=['CKAN', 'HydroShare'],
+                                         help='Type of dataset service being created (CKAN/HydroShare)'
+                                         '"<username>:<password>@<protocol>//<host>:<port>"')
+    services_create_dataset.add_argument('-c', '--connection', required=True, type=str,
+                                         help='The connection of the Service in the form '
+                                         '"<username>:<password>@<protocol>//<host>:<port>"')
+    services_create_dataset.add_argument('-p', '--public-endpoint', required=False, type=str,
+                                         help='The public-facing endpoint, \
+                                         if different than what was provided with the '
+                                         '--connection argument, of the form "<host>:<port>"')
+    services_create_dataset.add_argument('-k', '--apikey', required=False, type=str,
+                                         help='The API key, if any, required to establish a connection.')
+    services_create_dataset.set_defaults(func=services_create_dataset_command)
+
+    # tethys services create WPS
+    services_create_wps = services_create_subparsers.add_parser('WPS',
```

Comment:
@tylorbayer sorry I wasn't clear on this. Only uppercase WPS in the help strings, not in code. Please change `.add_parser('WPS', ...` back to `.add_parser('wps', ...`

Reply:
Resolved